components/NoteAction/NoteActionLike.tsx

```diff
@@ -35,8 +79,8 @@ const NoteActionLike = ({
           <HeartIcon width={18} height={18} />
         </motion.span>{" "}
         <Text lh="normal">
-          {note.reactions.length > 0 && note.event.pubkey === auth.pk
-            ? note.reactions.length
+          {reactions.length > 0 && event.pubkey === auth.pk
```

Comment:
Is this logic right? Shouldn't we always display the reaction count and not just for notes that the user made?

Reply:
Yeah we decided to only show like counts for your own notes